server/athenian/api/controllers/jira_controller.py

```diff
@@ -122,15 +121,10 @@ async def filter_jira_stuff(request: AthenianWebRequest, body: dict) -> web.Resp
         filt.account,
         request,
         JIRAFilterReturn.ISSUE_BODIES in return_ or JIRAFilterReturn.EPICS in return_,
-        request.sdb,
-        request.mdb,
-        request.pdb,
-        request.cache,
     )
     if filt.projects is not None:
-        projects = await resolve_projects(filt.projects, jira_ids.acc_id, request.mdb)
-        projects = {k: projects[k] for k in jira_ids.projects if k in projects}
-        jira_ids = JIRAConfig(jira_ids.acc_id, projects, jira_ids.epics)
+        projects = jira_conf.project_ids_map(filt.projects)
```

Comment:
no need to touch db again JIRAConfig has all the projects, we can compute the subset